cli/ballerina-cli/src/main/resources/cli-help/ballerina-build.help

```diff
@@ -43,24 +40,23 @@ OPTIONS
        --code-coverage
            Enable code coverage. This will analyze the line coverage of the
            source `.bal` files in the Ballerina package and add a section
-           to the testerina report with the code coverage details.
-           This feature is not supported with single file executions.
+           to the testerina report with the code coverage details. This feature is not supported with single file executions.
 
        --test-report
            Generate an HTML report containing the test results. Defaults to 'true'
            if code coverage is enabled.
 
        --debug
-           Run tests in remote debugging mode, only when the '--skip-tests' flag is not used.
+           Run tests in remote debugging mode only when the '--skip-tests' flag is not used.
 
        --observability-included
-           Package observability in the executable JAR file(s).
+           Include package observability in the executable JAR file(s).
 
        --cloud
-           Enable cloud artifact generation for cloud providers such as kubernetes.
+           Enable cloud artifact generation for cloud providers such as Kubernetes.
```

Comment:
shall we add supported values as well? 
`--cloud=docker` and `--cloud=k8s`

Reply:
Updated.